web/src/features/fba/slices/fireCentreHfiFuelTypesSlice.ts

```diff
@@ -0,0 +1,57 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { AppThunk } from 'app/store'
+import { logError } from 'utils/error'
+import { FireCentreHfiFuelsData, getHFIThresholdsFuelTypesForCentre } from 'api/fbaAPI'
+import { RunType } from 'features/fba/pages/FireBehaviourAdvisoryPage'
+
+interface State {
+  loading: boolean
```

Comment:
I don't see this loading state being used anywhere, you could remove from this slice and from `fireCentreTPIStatsSlice`.